lib/send-text.js

```diff
@@ -0,0 +1,15 @@
+// Copyright IBM Corp. 2016. All Rights Reserved.
+// Node module: strong-error-handler
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+
+var format = require('util').format;
```

Comment:
Use `const` - see http://loopback.io/doc/en/contrib/style-guide.html#variable-declarations

Reply:
Happy to do so. I think this is the case in quite a few places. I only saw `var` in the current codebase and so stuck with that.

Reply:
Yeah, we are conservative when it comes to code style changes and haven't upgrade the existing codebase for our new ES6 conventions. I know it can be confusing for new contributors.